resources/assets/less/bem/profile-info.less

```diff
@@ -170,6 +170,7 @@
   }
 
   &__title {
+    .link-white();
```

Comment:
the title should probably not be full width

![](https://s.myconan.net/2020-10/firefox_2020-10-07_14-53-22.png)